method/index.js

```diff
@@ -1,11 +1,17 @@
 const Init = require('./group/init')
 const AddMember = require('./group/add-member')
+const CreateLink = require('./link/create')
+const FindByGroupByFeedId = require('./link/find-group-by-feedid')
 
 module.exports = function Method (ssb, keystore, state) {
   return {
     group: {
       init: patient(Init(ssb, keystore, state)),
       addMember: patient(AddMember(ssb, keystore, state))
+    },
+    link: {
+      create: patient(CreateLink(ssb)),
+      findGroupByFeedId: FindByGroupByFeedId(ssb)
```

Comment:
does this need to be patient? I don't think so as the indexes query will make it patient for you